infra/aws/terraform/prow-build-cluster/resources/flux-system/ks-boskos.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+apiVersion: kustomize.toolkit.fluxcd.io/v1
+kind: Kustomization
+metadata:
+  name: boskos
+  namespace: flux-system
+spec:
+  interval: 5m0s
+  path: ./infra/aws/terraform/prow-build-cluster/resources/boskos
+  prune: false
```

Comment:
Thinking out loud, prune option (assuming as `kubectl apply --prune`) might be useful. What would it take to enable it and how it works for Flux?

Reply:
By default Flux does not remove any resources, so if any manifest has been removed, you still need to manually remove resources from your cluster. I was hesitating if flux should remove stuff, but I guess that should be ok.

Reply:
That’s similar to kubectl apply --prune, I think it should be safe and useful to enable it.